src/annotation_operators.ts

```diff
@@ -1,7 +1,13 @@
-import { ULabel, ULabelAnnotation, ULabelSubtask } from "..";
+import { Offset, ULabel, ULabelAnnotation, ULabelSpatialType, ULabelSubtask } from "..";
 
-//Given an annotation returns the confidence of that annotation
-export function get_annotation_confidence(annotation) {
+/**
+ * Returns the confidence of the passed in ULabelAnnotation.
+ * In the case of multiple classifications in the classification_payloads, returns the largest confidence value.
+ * 
+ * @param annotation ULabelAnnotation
+ * @returns The largest confidence value inside classification_payloads
+ */
+export function get_annotation_confidence(annotation: ULabelAnnotation) {
```

Comment:
would some of these functions make sense as class methods of `ULabelAnnotation`?

Reply:
Yes

Reply:
And its actually something that I think we should do soon. I do think that its out of the scope of this pr though, since I'm not adding this function, I just added the documentation for it.